	- explicitly use the host cc to compile c2sh and c2tmac
	  (using per-target local variable assignments is an elegant
	  way for not having to define explicit rules for the targets)
	- dont try setting owner and group of installed files
--- aufs2-util-130809.orig/Makefile	2009-08-13 14:59:49.000000000 +0200
+++ aufs2-util-130809/Makefile	2009-08-23 18:05:03.909726416 +0200
@@ -54,6 +54,11 @@ ${Dummy}: ${LibSoObj}
 ${LibSo}: ${Dummy}
 	ln -f $< $@
 
+c2sh c2tmac: CC = ${HOSTCC}
+c2sh c2tmac: CFLAGS="-I${KDIR}/include"
+c2sh c2tmac: LDFLAGS=
+c2sh c2tmac: CPPFLAGS=
+
 etc_default_aufs: c2sh aufs.shlib
 	${RM} $@
 	echo '# aufs variables for shell scripts' > $@
@@ -86,12 +91,12 @@ install_ulib: File = ${LibSo}
 install_ulib: Tgt = ${DESTDIR}/ulib
 install_sbin install_ubin install_man install_ulib: ${File}
 	install -d ${Tgt}
-	install -m 755 -o root -g root -p ${Opt} ${File} ${Tgt}
+	install -m 755 -p ${Opt} ${File} ${Tgt}
 install_etc: File = etc_default_aufs
 install_etc: Tgt = ${DESTDIR}/etc/default/aufs
 install_etc: ${File}
 	install -d $(dir ${Tgt})
-	install -m 644 -o root -g root -p -T ${File} ${Tgt}
+	install -m 644 -p -T ${File} ${Tgt}
 
 # do not inlcude install_ulib here
 install: install_man install_sbin install_ubin install_etc
